src/main/frontend/handler/editor.cljs

```diff
@@ -799,55 +798,54 @@
                                           (let [block (db/entity [:block/uuid block-id])]
                                             (seq (:block/_parent block)))))]
             (when-not (and has-children? left-has-children?)
-              (when block-parent-id
-                (let [block-parent (gdom/getElement block-parent-id)
```

Comment:
The main change is to use the editor's parent `ls-block` container instead of using the block-parent-id, the block-parent-id is something like `ls-block-uuid` that can be shared by multiple block dom nodes in the app.